pom.xml

```diff
@@ -22,7 +22,7 @@
     <raml-module-builder.version>33.2.6</raml-module-builder.version>
```

Comment:
raml-module-builder 33.2.8 is available.

Reply:
Hi Julian. The PR has draft status. I don't think that PR will be merged. I'll most likely close the PR and delete the branch/